.changeset/friendly-mugs-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/nextjs-kit": patch
+---
+
+DB-3781: Added ContentWithImage component to be used for a WP Post or Drupal Article
```

Comment:
The changeset messages will be appended directly to the changelog. We don't need the Jira ticket number here.